pincer/objects/guild/guild.py

```diff
@@ -885,6 +880,361 @@ async def delete(self):
         """
         await self._http.delete(f"guilds/{self.id}")
 
+    async def prune_count(
+        self,
+        days: Optional[int] = 7,
+        include_roles: Optional[str] = None
+    ) -> int:
+        """|coro|
+        Returns the number of members that
+        would be removed in a prune operation.
+        Requires the ``KICK_MEMBERS`` permission.
+
+        Parameters
+        ----------
+        days : Optional[:class:`int`]
+            Number of days to count prune for (1-30) |default| :data:`7`
+        include_roles : Optional[:class:`str`]
+            Comma-delimited array of Snowflakes;
+            role(s) to include |default| :data:`None`
+
+        Returns
+        -------
+        :class:`int`
+            The number of members that would be removed.
+        """
+        return await self._http.get(
+            f"guilds/{self.id}/prune?{days=}&{include_roles=!s}"
+        )["pruned"]
+
+    async def prune(
+        self,
+        days: Optional[int] = 7,
+        compute_prune_days: Optional[bool] = True,
+        include_roles: Optional[List[Snowflake]] = None,
+        reason: Optional[str] = None
+    ) -> int:
+        """|coro|
+        Prunes members from the guild. Requires the ``KICK_MEMBERS`` permission.
```

Comment:
```suggestion
        Prunes members from a guild, Requires the ``KICK_MEMBERS`` permission.
```

Reply:
Please try to fix grammar mistakes in your suggestions instead of introducing them 😅